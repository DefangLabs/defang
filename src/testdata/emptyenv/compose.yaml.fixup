{
  "emptyenv": {
    "build": {
      "context": ".",
      "dockerfile": "Dockerfile",
      "args": {
        "ARG2": ""
      }
    },
    "command": null,
    "deploy": {
      "replicas": 1,
      "resources": {},
      "placement": {}
    },
    "entrypoint": null,
    "environment": {
      "ENV1": null,
      "ENV2": ""
    },
    "networks": {
      "default": null
    }
  }
}