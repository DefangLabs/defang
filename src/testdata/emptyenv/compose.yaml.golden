name: emptyenv
services:
  emptyenv:
    build:
      context: .
      dockerfile: Dockerfile
      args:
        ARG2: ""
    environment:
      ENV1: null
      ENV2: ""
    networks:
      default: null
networks:
  default:
    name: emptyenv_default
